README.md

```diff
@@ -52,7 +53,44 @@ that everything is up to date, run the following commands:
 - `flake8` to check for style warnings, errors etc.
 - `python manage.py test` to run tests.
 
-# Docker
+# Reading from the Arduino
+
+There is a separate application for reading from the Arduino in the `sensors`
+folder, if you want to read some data from the Arduino you can run it from a
+terminal and get a single reading from the sensors.
+
+``` python
+from sensors.arduino import Arduino
+arduino = Arduino()
+arduino.read()
+# ('2122.39', '26.31')
+```
+
+# Using Celery
+
+We are using Celery for asynchronously being able to query the Arduino and add
+the results to the database. For this you need to have RabbitMQ installed as it
+is our broker, then to actually run Celery and RabbitMQ you need to run the
+following two commands in two separate terminal windows:
+
+1. `rabbitmq-server`
+2. `celery -A moccapi worker -l info -B`
+
+Running the commands from Python you need to start the Python Console from
+Pycharm via `Tools -> Python Console`, then run the following:
+
+``` python
```

Comment:
Reminder to myself to remove this section as it doesn't exist in the code anymore :man_facepalming: